src/main/java/com/hotels/housekeeping/model/DefaultLegacyReplicaPath.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2016-2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.housekeeping.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+import org.hibernate.envers.Audited;
+
+@Entity
+// @Converter doesn't work with @Audited until https://hibernate.atlassian.net/browse/HHH-9042 is released
+@Audited
+@Table(schema = "housekeeping", name = "legacy_replica_path",
```

Comment:
the schema name has changed here, I've still not looked at CT but we shouldn't change it yet until the separation is completely done so we can define our own entity for CT. If this is not possible we must make clear in CHANGELOG.md that after this version is installed they must run the vacuum tool and ditch the old DB

Reply:
This is a default implementation, you'll see in the CT PR that it extends this class and sets the table name to "circus_train" so it's still backwards compatible.